pages/src/source/tutorials/schematics.md

```diff
@@ -393,3 +404,79 @@ Put the [deco controller](../../source/items/decocontroller) somewhere in the sc
 {% endversion %}
 
 ![Upgradable Decos](../../assets/images/tutorial/upgradabledecos.png)
+
+
+### How to use custom mineshafts in style packs?
+
+The size must be 9 x 4 x 9 blocks. Rotation must match the Mine in the Style Pack. Use an existing style pack as a template along with the [scan tool](../../source/items/scantool) to create the blueprints.
```

Comment:
I copied this part about mineshaft rotation verbatim, but I'm not sure it's correct.  It seems to be stating that you should rotate schematics to match the default placement in the mine, whereas I thought the rule was to not rotate anything from the defaults and let the code sort it out.  I haven't done any testing on this myself, though, so idk which is correct.

Reply:
You may indeed not rotate the shafts at all